cmake/FindDev.cmake

```diff
@@ -28,7 +28,7 @@ else()
 		find_library(Dev_${L}_LIBRARY
 			NAMES ${l}
 			PATHS ${CMAKE_LIBRARY_PATH}
-			PATH_SUFFIXES "lib${l}" "${l}" "lib${l}/Release" 
+			PATH_SUFFIXES "lib${l}" "${l}" "lib${l}/Debug" "lib${l}/Release" 
```

Comment:
Is there any reason we don't check the "/Debug" folder?

On Mac OSX the library is put in either Debug or Release folder.